src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -45,12 +46,20 @@ export class ListingCreateComponent implements OnInit {
                 private readonly listingNotesConfig: ListingCreateNotesConfiguration) {
 
         this.store.select(getHearingPartsError).subscribe((error: any) => {
-            this.errors = error.message;
+            if (error !== undefined ) {
+                this.errors = error.message;
+            }
+        });
+        this.store.select(fromJudges.getJudges).withLatestFrom(
```

Comment:
Why use the 'withLatestFrom' here? I don't see any merging if judges and casetypes inside the subscriber. It could be two separate store selects, not chained with 'withlatestFrom'

Reply:
because I want them both to be available in the subscriber, simple pipe won't gonna work